pkg/asset/installconfig/azure/validation_test.go

```diff
@@ -0,0 +1,173 @@
+package azure
+
+import (
+	"net"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+
```

Comment:
nit: extra space

Reply:
You want me to add an extra space, or you want me to remove an extra space? where?

Reply:
remove the extra line here.